ui/raidboss/data/07-dt/trial/zoraal-ja.ts

```diff
@@ -531,82 +540,122 @@ const triggerSet: TriggerSet<Data> = {
   timelineReplace: [
     {
       'locale': 'de',
-      'missingTranslations': true,
       'replaceSync': {
         'Fang': 'Reißzahn',
+        'Shadow of Tural': 'Schatten Turals',
```

Comment:
Here and elsewhere, in both this and the FRU translation block, as with the UCoB PR you just did, can you capitalize "of", "an", "the", etc? I know some of these lines are pre-existing from previous PRs, but it should be okay to update those alongside your new additions.